subtree.js

```diff
@@ -43,9 +43,21 @@ module.exports = {
       return elem;
     }
 
+    // if it's an iframe, copy the children of contentDocument.
+    if(iframe) {
```

Comment:
Nitpick: As the only code that runs is the entirety of the if-clause below, I'd prepend this to the if condition in the next line.

Reply:
fixed in newest commit